Jellyfin.Plugin.PlaybackReporting/Pages/breakdown_report.js

```diff
@@ -265,8 +265,12 @@ define(['libraryMenu'], function (libraryMenu) {
                 process_click();
 
                 function process_click() {
-                    var days = parseInt(weeks.value) * 7;
-
+                    //Need to implement a better method to select all data instead of 18250 days = 50 years. 
+                    var days = 18250;
+                    if (parseInt(weeks.value) != -1){
+                        var days = parseInt(weeks.value) * 7;
```

Comment:
```suggestion
                        days = parseInt(weeks.value) * 7;
```

Reply:
For some reason, this one was not applied.

FYI, you can combine suggested changes with `Add suggestion to batch`.